indexer/src/main/java/au/org/aodn/esindexer/utils/MapperUtils.java

```diff
@@ -410,4 +422,90 @@ public static Optional<Contacts> mapContactInfo(List<CIContactPropertyType2> con
         }
     }
 
+    public static List<ContactsModel> mapContactsFromOrg(CIResponsibilityType2 ciResponsibility, CIOrganisationType2 organisation) {
+
+        Optional<MapperUtils.Contacts> org = MapperUtils.mapContactInfo(organisation.getContactInfo());
+        return new ArrayList<>(organisation
+                .getIndividual()
+                .stream()
+                .map(individual -> {
+                    ContactsModel invContactsModel = ContactsModel.builder().build();
+                    invContactsModel.setName(MapperUtils.mapContactsName(individual));
+                    invContactsModel.setPosition(MapperUtils.mapContactsPosition(individual));
+                    invContactsModel.setRoles(MapperUtils.mapContactsRole(ciResponsibility));
+                    invContactsModel.setOrganization(organisation.getName().getCharacterString().getValue().toString());
+
+                    Optional<MapperUtils.Contacts> inv = MapperUtils.mapContactInfo(individual.getCIIndividual().getContactInfo());
```

Comment:
always not null ? `individual.getCIIndividual().getContactInfo()`, if not , may need 2 steps null checks, `individual.getCIIndividual() != null` and `individual.getCIIndividual().getContactInfo() != null`

Reply:
should name `individualContactInfo` instead of `inv` for clarity